storybook/src/pages/amsterdam/NavigationPage/NavigationPage.stories.tsx

```diff
@@ -315,28 +326,28 @@ export const SubnavigationPage: StoryObj = {
           </Grid.Cell>
           <LinkGroupGridCells linkGroups={burgerzakenLinks.slice(0, 6)} />
           <Grid.Cell span={{ narrow: 4, medium: 4, wide: 5 }} start={{ narrow: 1, medium: 1, wide: 2 }}>
-            <Heading className="ams-mb-s" level={2} size="level-4">
+            <Heading className="ams-mb-s" level={3}>
```

Comment:
I think these should be level 2? Or is 'L2 Paragraaf titel' the parent of all these headings? In that case, they should all be level 3, not just these 4. 

You can use something like the HeadingsMap Chrome plugin to check the heading hierarchy.

Reply:
I'm not sure I follow. I'm talking about the levels here, not the sizes.

The current hierarchy is wrong:

<img width="344" height="584" alt="image" src="https://github.com/user-attachments/assets/b8a71bbe-7ad3-4901-b4f7-0d572aeafc6a" />

'14 kades en bruggen hersteld ...' is not a child of 'Geboorte en erkenning kinderen', etc.

I'm not sure if everything here should be level 2, of if 'L2 Paragraaf titel' is meant as a parent. In that case, it should be something like:

```
L1 Vermakelijkhedenretributie vanaf 2023
-- L2 Paragraaf titel
---- L3 Paspoort en identiteitskaart
---- L3 Rijbewijs
```

And is 'Wat vindt u van de inzameling van grof afval in Amsterdam?' also part of 'L2 Paragraaf titel'? Or is that level 2 again?